examples/go-tracepoint-counter/main.go

```diff
@@ -102,21 +108,27 @@ func main() {
 	// retrieve and report on the number of kill -SIGUSR1 calls
 	index := uint32(0)
 	ticker := time.NewTicker(1 * time.Second)
-	defer ticker.Stop()
-	for range ticker.C {
-		var stats []Stats
-		var totalCalls uint64
+	go func() {
```

Comment:
What was the reason for putting this in a go-routine?

Reply:
The tracepoint programs weren't being unloaded. The defer statements were being skipped on a <CTRL-C>. I just ported over the XDP and TC pattern.

Reply:
Ok I'm alright with that... I feel like that pattern also _could_ result in a break when `ctrl-c`ed at the right time :)